Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -21,10 +21,9 @@ public protocol ChatMessageContentViewDelegate: AnyObject {
     func messageContentViewDidTapOnThread(_ indexPath: IndexPath?)
 
     /// Gets called when quoted message view is tapped.
-    /// - Parameter indexPath: The index path of the cell displaying the content view. Equals to `nil` when
-    /// the content view is displayed outside the collection/table view.
-    func messageContentViewDidTapOnQuotedMessage(_ indexPath: IndexPath?)
-
+    /// - Parameter quotedMessage: The quoted message which was tapped.
+    func messageContentViewDidTapOnQuotedMessage(_ quotedMessage: ChatMessage)
```

Comment:
A breaking change here.

Reply:
Yes, it is a breaking change, and we will mention this in the Changelog. But this breaking change is required. The previous implementation was wrong, so nobody should be even using this. If they are, it is a bug, since the IndexPath was incorrect, and does not make any sense. If the tapped quoted message is not available in the UI, there is no indexPath, so what we are interested in here is the message itself, not the indexPath.